packages/analytics-browser/src/browser-client.ts

```diff
@@ -261,16 +274,27 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
   }
 
   async process(event: Event) {
+    let shouldTrackNewCampaign = false;
+    let shouldSetSessionId = false;
+    if (this.webAttribution) {
```

Comment:
This is the replacement of `.?` for the test coverage purpose. Jest test cannot catch the branch coverage for `.?`.